Automattic/pocket-casts-android
# Move help menu to profile screen

## Description
This moves `Help & feedback` menu from `Settings` to `Profile` screen.
Internal ref: p1701065643945509-slack-C029BMLUGRX

## Testing Instructions
1. Launch the app
2. Go to `Profile` tab
3. Scroll to bottom
4. Notice there's a `Help & feedback` menu
5. Tap on the menu
6. ✅ Notice that `Help & feedback` screen is displayed
8. Tap on `Help & feedback` menu items (`Logs`, `Status Page`)
9. ✅ Notice that corresponding menu item screen is displayed
10. Tap on the back button till to you return to the `Profile` screen
11. ✅ Notice that back functions properly
12. Tap `Settings` (gear icon) on `Profile` screen
13. ✅ Notice that `Help & feedback` is not present on the screen 

**Automotive app**

1. Go to `Settings` -> `About` 
2. Tap `Logs` menu
3. ✅ Notice that `Logs` screen is displayed 
4. Tap back on `Logs` screen 
5. ✅ Notice that back functions properly


## Screenshots or Screencast 

1|2|3
---|---|---
<img width=200 src ="https://github.com/Automattic/pocket-casts-android/assets/1405144/702f61bf-7314-46e4-aa21-380628fbab5d"/>|<img width=200 src ="https://github.com/Automattic/pocket-casts-android/assets/1405144/eb6ca519-eb59-47fd-92f2-cc670d1b7e4c"/>|<img width=200 src ="https://github.com/Automattic/pocket-casts-android/assets/1405144/588e51cb-4279-4dd4-8e2d-20e5356f7ce7"/>

<img width=300 src ="https://github.com/Automattic/pocket-casts-android/assets/1405144/a7131fe2-c430-48e4-8f30-1330d6486071"/>


Since I replaced the menu icon, I also checked that it is displayed properly on the watch app (in the preview)

<img width=100 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/730601f7-eca4-41f8-8369-8433a4578842"/>


## Checklist
 
#### I have tested any UI changes...
